orc8r/cloud/order_go_imports.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash                                                                      
+                                                                                 
+# remove all blank lines in go 'imports' statements,                             
+# then sort with goimports                                                       
+                                                                                 
+if [ $# != 1 ] ; then                                                            
+  echo "usage: $0 <filename>"                                                    
+  exit 1                                                                         
+fi                                                                               
+                                                                                 
+sed -i ''  '
+  /^import/,/)/ {
+    /^$/ d
+  }
+' "$1"
+
+goimports -w -local magma/ $1
```

Comment:
How is the sort supposed to be in this case?  go library modules,  3rd party, magma modules (?)

like here ?

![image](https://user-images.githubusercontent.com/16157139/133690541-4446fd5f-8a06-4ea6-81ce-7a9d6dd97139.png)

Reply:
Yes, if you remove empty lines between import blocks first, goimports sorts them correctly in this order and then re-inserts the empty lines.

Reply:
ohh , it is going to be   goLibs, magma then 3rd party libs 

because the `magma` prefix at ```goimports -w -local magma/```